src/com/lftechnology/java/training/srijan/basics/coreapp/ui/EmployeeView.java

```diff
@@ -0,0 +1,397 @@
+
+package com.lftechnology.java.training.srijan.basics.coreapp.ui;
+
+import com.lftechnology.java.training.srijan.basics.coreapp.controller.EmployeeController;
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.Employee;
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.SearchEmployee;
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.UserRole;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class EmployeeView {
+
+	private static final Logger LOGGER =
+		Logger.getLogger(EmployeeView.class.getName());
+	List<Employee> employees;
+	Employee employee;
+	Employee e = new Employee();
+	EmployeeController employeeControl = new EmployeeController();
+	private int choiceUserRole;
+	private int userStatus;
+	private String username;
+	private String fullname;
+	private String department;
+	private String address;
+	private String password;
+
+	public EmployeeView() {
+		this.choiceUserRole = 0;
+		this.userStatus = 0;
+		this.username = "";
+		this.fullname = "";
+		this.department = "";
+		this.address = "";
+		this.password = "";
+	}
+
+	/**
+	 * <p>Takes employee information from user</p>
+	 * 
+	 * @param e
+	 * @return
+	 * @author srijan
+	 */
+	public Employee inputUserInfo(Employee e, Scanner input) {
+
+		Boolean checkUserValidity;
+		try {
+			input = new Scanner(System.in);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "input". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1226)